components/net/Cargo.toml

```diff
@@ -73,6 +74,7 @@ futures = { version = "0.3", features = ["compat"] }
 tokio-test = "0.4"
 tokio-stream = { version = "0.1", features = ["net"] }
 hyper = { workspace = true, features = ["full"] }
+tempfile = "3"
```

Comment:
is this one still used? I thought we removed the filesystem testing?

Reply:
Ah, well spotted. The "tempfile" crate is used in some existing code in Servo, but I don't believe it needs to be included in the `components/net` crate. I've removed it.